db/thrman.c

```diff
@@ -471,7 +471,7 @@ static int thrman_check_threads_stopped_ll(void *context)
         all_gone = 1;
 
     /* if we're exiting then we don't want a schema change thread running */
-    if (db_is_stopped() && 0 != thr_type_counts[THRTYPE_SCHEMACHANGE])
```

Comment:
why?

Reply:
This logic didn't change as part of this PR, but I also question it.